internal/server/pending_cases.go

```diff
@@ -33,7 +33,7 @@ func pendingCases(client PendingCasesClient, tmpl Template) Handler {
 			return err
 		}
 
-		criteria := sirius.Criteria{}.Filter("status", "Pending").Page(getPage(r)).Sort("receiptDate", sirius.Ascending)
+		criteria := sirius.Criteria{}.Filter("status", "Pending").Page(getPage(r)).Sort("workedDate", sirius.Descending)
```

Comment:
This should be unlikely to occur with real data, but it would probably be worth having a backup sort in case there are multiple unworked cases.
```suggestion
		criteria := sirius.Criteria{}.Filter("status", "Pending").Page(getPage(r)).Sort("workedDate", sirius.Descending).Sort("receiptDate", sirius.Ascending)
```